### PR TITLE
Add ppc64le to cross targets

### PR DESCRIPTION
**Pull request by vdemeester**
# Changes

ppc64le is the POWER architecture 🥀

Closes #1037

Signed-off-by: Vincent Demeester <vdemeest@redhat.com>

/cc @snehlatamohite

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [x] Run the code checkers with `make check`
- [x] Regenerate the manpages, docs and go formatting with `make generated`
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

```release-note
cross compiling for the POWER architecture (ppc64le)
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -55,6 +55,10 @@ arm64:
 s390x:
 	GOOS=linux GOARCH=s390x go build -mod=vendor $(LDFLAGS) -o bin/tkn-linux-s390x ./cmd/tkn
 
+.PHONY: ppc64le
+ppc64le:
+	GOOS=linux GOARCH=ppc64le go build -mod=vendor $(LDFLAGS) -o bin/tkn-linux-s390x ./cmd/tkn
```
Comment (yselkowitz) [inline Makefile:60]: Copy-paste error, you've got a stray s390x at the end of that.

Comment (vdemeester) [reply Makefile:60]: indeed :sweat_smile:

File: README.md [hunk 2]
```diff
@@ -73,6 +73,18 @@ Download the latest binary executable for your operating system:
   sudo tar xvzf tkn_0.10.0_Linux_s390x.tar.gz -C /usr/local/bin/ tkn
   ```
 
+* Linux IBM P
+
+  *To complete on 0.11.0 release*
```
Comment (danielhelfand) [inline README.md:78]: Nit:

```suggestion
  *Will be available with tkn v0.11.0 release*
```

**Pull request merged**